releasenotes/notes/0.19/add-qubit-subset-to-bip-mapper-e1c6234d04484d58.yaml

```diff
@@ -2,7 +2,7 @@
 features:
   - |
     Introduced a new option ``qubit_subset`` to the constructor of
-    :class:`qiskit.transpiler.passes.BIPMapping`.
+    :class:`.BIPMapping`.
```

Comment:
Is there any difference with this?

Reply:
It'll only display `BIPMapping` rather than `qiskit.transpiler.passes.BIPMapping`, but I don't know why I didn't just prefix it with a `~`.  In general, I shortened the output of everything down to the necessary information (mostly `Class.method` and occasionally `module.function` or `module.Class`), because that makes it easier to read and you can always click the link to get the full path if you need it.